src/parser/warrior/arms/modules/features/RageUsage.js

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import { Doughnut as DoughnutChart } from 'react-chartjs-2';
+
+import SPELLS from 'common/SPELLS';
+
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+
+import Analyzer from 'parser/core/Analyzer';
+import StatisticsListBox from 'interface/others/StatisticsListBox';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import StatisticWrapper from 'interface/others/StatisticWrapper';
+
+const CHART_SIZE = 100;
+
+const LIST_OF_RAGE_SPENDER = [
+    SPELLS.CLEAVE_TALENT.id,
+    SPELLS.WHIRLWIND.id,
+    SPELLS.HAMSTRING.id,
+    SPELLS.MORTAL_STRIKE.id,
+    SPELLS.EXECUTE.id,
+    SPELLS.SLAM.id,
+    SPELLS.IMPENDING_VICTORY_TALENT.id,
+    SPELLS.REND_TALENT.id,
+];
+
+class RageUsage extends Analyzer {
```

Comment:
I'm not sure what the *general consensus* on doughnut charts are, some like it, some hate it (\*cough\* emallson \*cough\*), and some like me are kinda neutral. I think they are visually pleasing, but the fact that they show pretty much the same thing as the Resource tab (if implemented) and the fact that they can become very cluttered if many spells are displayed, doesn't really tip the scales in their favor. Related discussion in #2333 .

Reply:
I could get rid of this part and implement the Ressource tab later on if you prefer

Reply:
Unless someone else reviews this and decides you should delete it, I'd say keep it 😄 Just pointing the facts out, that it's pretty much the same information.

Reply:
Alright, i might change this in the futur anyway to avoid having too much informations not that meaningfull on the statistic part but i leave it that way for now

Reply:
as much as i hate doughnut charts, I'm not gonna hold this PR up for including one :P. when you get around to implementing the Resources tab, this can get the axe